config.schema.yaml

```diff
@@ -155,6 +155,18 @@ alertmanager_accounts:
         secret: false
         default: ""
 
+router_sensor:
+  description: "Openstack Router sensor specific settings."
+  type: "object"
+  required: false
+  additionalProperties: false
+  properties:
+    cloud_account:
+      type: "string"
+      description: "cloud account to use - set in clouds.ymal"
+      default: "dev"
+      required: true
```

Comment:
Can we make this a global config option? Since I can't think of a case where we'll have a ST2 instance serving prod and dev

This avoids us having to set it per-sensor going forward too